modules/gitops-release-notes-1-16-0.adoc

```diff
@@ -0,0 +1,105 @@
+
+[id="fixed-issues-1-16-0_{context}"]
+== Fixed issues
+
+* Before this update, when installing a namespace-scoped instance of {gitops-shortname}, the `argocd-redis ServiceAccounts` were assigned the `nonroot-v2 SecurityContextConstraints` (SCC), which provided more privileges than the standard `restricted-v2` SCC, which may lead to potential security risks. With this update, the namespace-scoped and cluster-scoped instances of {gitops-shortname} enforce the use of the `restricted-v2` SCC for the `argocd-redis ServiceAccounts`. This change enhances security compliance by minimizing unnecessary privileges. link:https://issues.redhat.com/browse/GITOPS-6236[GITOPS-6236]
+
+* Before this update, the on-deployed trigger in the Argo CD notification-controller could incorrectly send a success notification while the application was still in the `progressing` state. This issue arose from the way Argo CD handled application status updates. With this update, a new timestamp field, `status.health.lastTransitionTime`, has been introduced in the application status to address this issue. This field captures the last health status change, enabling the on-deployed trigger to send notifications only after a stable transition, to prevent false-positive alerts. link:https://issues.redhat.com/browse/GITOPS-3699[GITOPS-3699]
+
+* Before this update, during an upgrade, the `argocd-redis-ha-configmap`, `argocd-redis-ha-health-configmap`, and the Redis HA `StatefulSet` were not correctly updated. This led to Redis HA pods encountering an `AUTH` error.
+With this update, the {gitops-shortname} Operator correctly updates the Redis HA configmaps and `StatefulSet` during an upgrade process. As a result, Redis HA pods are now prevented from entering an `AUTH` error state post-upgrade. link:https://issues.redhat.com/browse/GITOPS-5975[GITOPS-5975]
+
+* Before this update, any changes to the `serviceAccountName` and `serviceAccount` fields in the Redis deployment were not reconciled by the {gitops-title} Operator. With this update, this issue is fixed by ensuring that any unintended changes to these fields are reset to their expected value, `<argocd-instance-name>-argocd-redis`. link:https://issues.redhat.com/browse/GITOPS-6032[GITOPS-6032]
+
+* Before this update, Argo CD relied solely on the `sub claim` for user identification, which could be non-deterministic with Dex and cause unexpected RBAC policy failures.With this update, Argo CD identifies users in the following order:
++
+--
+** First, it checks the `federated_claims.user_id` when Dex is the identity provider.
+** Second, if federated claims are unavailable or empty, it falls back to the sub claim.
+--
++
+--
+With this update, this issue is fixed. This change ensures RBAC policies are based on actual user identifiers, such as email addresses rather than encoded values, improving clarity and consistency.
```

Comment:
```suggestion
With this update, this issue is fixed. This change ensures RBAC policies are based on actual user identifiers, such as email addresses rather than encoded values.
```

IMO this can be removed, up to you

Reply:
Fixed.